docs/assistant/connect-to-openai.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[assistant-connect-to-openai]]
+= Connect to OpenAI
+
+This page provides step-by-step instructions for setting up an OpenAI connector for the first time. This connector type enables you to leverage OpenAI's large language models (LLMs) within {kib}. You'll first need to create an OpenAI API key, then configure the connector in {kib}. 
+
+
+[discrete]
+== Configure OpenAI
+
+[discrete]
+=== Select a model
+
+Before creating an API key, you must choose a model. Refer to the https://platform.openai.com/docs/models/gpt-4-turbo-and-gpt-4[OpenAI docs] to select a model. Take note of the specific model name (for example `gpt-4-turbo`); you'll need it when configuring {kib}.
+
+NOTE: `GPT-4 Turbo` offers increased performance. `GPT-4` and `GPT-3.5` are also supported. 
```

Comment:
NOTE: For best performance we recommend gpt-4o. Alternatively you can use gpt-4-turbo, but at a minimum, gpt-4-32k can be used. 
--


<br class="Apple-interchange-newline">
--


<br class="Apple-interchange-newline">


<br class="Apple-interchange-newline">or